build-php-brew.sh

```diff
@@ -1,13 +1,21 @@
 set -ex
 
-brew install bison openssl@1.1 pkg-config coreutils autoconf
```

Comment:
I think this is still needed - I tried it locally and failed without `bison` and `pkg-config`

Reply:
hmm, yeah, I forgot to test it without all these libraries.
I've just tried and it seems the only required libs for me are `autoconf` and `pkg-config`.
I don't know why in my case I don't need `bison`, but just in case I returned all the packages except openssl

Reply:
Bison was needed in my case by the configure script, not the build script so maybe it's not needed anymore after the changes.


In any case, since this is only needed on the build machine, we can leave it as is. Great work!